src/schema/v2/me/index.ts

```diff
@@ -221,6 +221,9 @@ export const meType = new GraphQLObjectType<any, ResolverContext>({
     profession: {
       type: GraphQLString,
     },
+    otherRelevantPosition: {
+      type: GraphQLString,
```

Comment:
@olerichter00 you might want to add a description here.

Reply:
Good idea 👍